crowbar_framework/app/views/barclamp/glance/_edit_attributes.html.haml

```diff
@@ -16,6 +16,8 @@
       %legend
         = t(".store.file_header")
       = string_field :filesystem_store_datadir
+      .alert.alert-info{ "data-show-for-clusters-only" => "true", "data-elements-path" => "glance-server" }
+        = t('.store.file_ha_info')
```

Comment:
Why? Can't you embed the alert in a div that would only be displayed if the selection is File?